src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -50,8 +50,9 @@ static NodeFunctionInvoker()
             Initialize();
         }
 
-        internal NodeFunctionInvoker(ScriptHost host, BindingMetadata trigger, FunctionMetadata functionMetadata, Collection<FunctionBinding> inputBindings, Collection<FunctionBinding> outputBindings)
```

Comment:
There are also host level logs that we're writing to the host log files that we might eventually want to happen only on primary, but since we're not log streaming those currently, it's not an issue yet.